spec/cancan/ability_spec.rb

```diff
@@ -361,6 +361,20 @@ class A; include B; end
     expect(@ability.can?(:read, A)).to be(true)
   end
 
+  it "allows to check ability in a decorated class" do
+    class ADecorator < SimpleDelegator
+      def kind_of?(klass)
```

Comment:
We shouldn't have to override kind_of. I'm not sure we want to support things that override kind_of. It's inflexible. I can look at what's required, but draper should not be overriding this, as automatic delegation is an option not a default.